src/cli/serve/serve.js

```diff
@@ -161,14 +156,9 @@ export default function (program) {
       [getConfigPath()]
     )
     .option('-p, --port <port>', 'The port to bind to', parseInt)
-    .option('-q, --quiet', 'Deprecated, set logging level in your configuration')
     .option('-Q, --silent', 'Prevent all logging')
     .option('--verbose', 'Turns on verbose logging')
```

Comment:
`Prevent all logging` and `Turns on verbose logging` are not exactly true with the new logging system, as those cli options just impact the root logger's level. E.g if `silent` is `true` but the logging configuration defines sub loggers with higher levels, those loggers will still be used for their respective contexts.

Now, do we think this is an issue, because if we do, fixing it to mimic the legacy behavior is not necessarily trivial, and would require to have the logging system use those `silent` and `verbose` options. Like, when either is present, it would not register any sub logger. This could potentially have serious impacts though (thinking about audit logging for example).

Ihmo the pragmatic solution is to just change the options description:

```
.option('-Q, --silent', 'Sets the root logger level to off')
.option('--verbose', 'Sets the root logger level to all')
``` 

wdyt?

Reply:
> Ihmo the pragmatic solution is to just change the options description:

My +1. We don't have to be backward compatible with the legacy system because they have a completely different configuration model and sub-loggers do not match the legacy system setup.